save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/DiktatDemoController.kt

```diff
@@ -0,0 +1,41 @@
+package com.saveourtool.save.sandbox.controller
+
+import com.saveourtool.save.configs.ApiSwaggerSupport
+import com.saveourtool.save.demo.diktat.DiktatDemoMode
+import com.saveourtool.save.demo.diktat.DiktatDemoResult
+import com.saveourtool.save.demo.diktat.DiktatDemoTool
+import com.saveourtool.save.sandbox.service.DiktatDemoService
+import com.saveourtool.save.utils.*
+
+import io.swagger.v3.oas.annotations.tags.Tag
+import io.swagger.v3.oas.annotations.tags.Tags
+import org.springframework.web.bind.annotation.*
+import reactor.core.publisher.Mono
+
+/**
+ * Controller for diktat-demo
+ */
+@ApiSwaggerSupport
+@Tags(
+    Tag(name = "diktat-demo"),
+)
+@RestController
+@RequestMapping("/diktat-demo/api")
```

Comment:
```suggestion
@RequestMapping("/api/diktat-demo")
```

Reply:
Later it is planned to move `diktat-demo` and other demos to a new service so the endpoint would be `/demo/api/diktat`